src/core/importType.js

```diff
@@ -29,10 +29,11 @@ function isExternalPath(path, name, settings) {
 }
 
 function isSubpath(subpath, path) {
-  const normSubpath = subpath.replace(/[/]$/, '')
+  const normSubpath = subpath.replace(/\/$/, '')
   if (normSubpath.length === 0) {
     return false
   }
+  path = path.replace(/\\/g, '/')
```

Comment:
perhaps instead of handling both cases, we should use `path.sep`, so that it handles file paths according to the filesystem you're on?

Reply:
I'm not sure that if `path.sep` works, but I guess no harm we just simply normalize the path?

Reply:
in that case, would node's `path.normalize` work?

Reply:
I don't think `path.normalize` work

```text
require('path').normalize('E:/path\\to/')
'E:\\path\\to\\'
```